kbs/src/jwe.rs

```diff
@@ -43,23 +121,277 @@ pub fn jwe(tee_pub_key: TeePubKey, payload_data: Vec<u8>) -> Result<Response> {
 
     let rsa_pub_key =
         RsaPublicKey::new(n, e).context("Building RSA key from modulus and exponent failed")?;
-    let sym_key: &[u8] = aes_sym_key.as_slice();
-    let wrapped_sym_key = rsa_pub_key
-        .encrypt(&mut rng, Pkcs1v15Encrypt, sym_key)
+    let padding = Oaep::new::<Sha256>();
+    let encrypted_key = rsa_pub_key
+        .encrypt(&mut rng, padding, aes_sym_key.as_slice())
         .context("RSA encrypt sym key failed")?;
 
-    let protected_header = json!(
-    {
-       "alg": RSA_ALGORITHM.to_string(),
-       "enc": AES_GCM_256_ALGORITHM.to_string(),
-    });
+    Ok(Response {
+        protected,
+        encrypted_key,
+        iv: iv.into(),
+        ciphertext: payload_data,
+        aad: None,
```

Comment:
set aad to None is deliberated? why?

Reply:
Yes. It's specified in https://datatracker.ietf.org/doc/html/rfc7516#section-5.1 step 14